starport_template/lib/pages/routing_page.dart

```diff
@@ -25,7 +25,7 @@ class _RoutingPageState extends State<RoutingPage> {
       if (!mounted) {
         return;
       }
-      if (store.wallets.isEmpty) {
+      if (store.wallets.isNotEmpty) {
```

Comment:
Why did we update this condition?

Reply:
good catch! I always forget about changing this back